Code spaces dev tools (#18893)
Azure/azure-sdk-for-js: common/tools/dev-tool/README.md

The `dev-tool about` command will print some information about how to use the command. All commands additionally accept the `--help` argument, which will print information about the usage of that specific command. For example, to show help information for the `resolve` command above, issue the command `dev-tool package resolve --help`.

> ## Usage in this repository

Reviewer: ```suggestion
### Publishing samples
```

> *To build (transpile) the sample-dev folder of a service's SDK*:

Reviewer: ```suggestion
*To create/update the samples folder of a service's SDK*:
```

1. Change the terminal to the service's `samples-dev` folder.  

> 1. Run `dev-tool samples publish --force samples-dev`. The generated files are placed in the `samples` folder.

Reviewer: ```suggestion
1. Run `dev-tool samples publish --force samples-dev`.
```
